parsil/src/placeholders.rs

```diff
@@ -43,9 +43,16 @@ impl<'a, C: ContextProvider> PlaceholderValidator<'a, C> {
         Ok(())
     }
 
-    /// Ensure that all the placeholders have been used, and return the largest
-    /// one found.
+    /// Ensure that the number of parameters matches the number of placeholders,
+    /// and return the largest one found.
```

Comment:
“Ensure that all the placeholders have been used” is still true, right?

Reply:
I agree with with that, I just don't understand why you removed the part of the comment I mentioned.

Reply:
Perfect